e2e/cypress/e2e/navigationdropdown.cy.js

```diff
@@ -5,17 +5,36 @@ describe("Dropdown navigation on small screens", () => {
   });
 
   it("should show the profile menu", () => {
-    cy.get('[data-cy="toggle-text"]').contains("Profilmenu").click();
+    cy.get('[data-cy="menu-title"]').contains("Profilmenu").click();
     cy.get('[data-cy="mobile-link-myLibraries"]').click();
-    cy.get('[data-cy="mobile-link-myLibraries"]')
-      .parent()
-      .should("have.css", "background-color", "rgb(51, 51, 255)");
+    cy.get('[data-cy="mobile-link-myLibraries"]').should(
+      "have.css",
+      "background-color",
+      "rgb(242, 242, 242)"
+    );
     cy.get('[data-cy="mobile-link-myLibraries"]').should(
       "have.attr",
       "href",
-      "/profil/mine-biblioteker"
+      "/mine-biblioteker"
     );
-    cy.get('[data-cy="mobile-link-myLibraries"]').click();
-    cy.get(".dropdown-menu").should("not.be.visible");
+  });
+
+  it("close dropdown when clicking on selected item", () => {
+    cy.get('[data-cy="menu-title"]').contains("Profilmenu").click();
+    // click twice, since redirect is not triggered and thus page doesnt reload with new closed menu
+    cy.get('[data-cy="mobile-link-myLibraries"]').click().click();
+    cy.contains("ul").should("not.be.visible");
```

Comment:
Tror der står to gange .click()

Reply:
ja, der er en kommentar foran: /
/ click twice, since redirect is not triggered and thus page doesnt reload with new closed menu
måske kunne man intercepte den...

Reply:
hov sorry læste ikke kommentaren😬

Reply:
hov sorry læste ikke kommentaren😬